src/spyglass/lfp/lfp_merge.py

```diff
@@ -0,0 +1,53 @@
+import datajoint as dj
+from spyglass.lfp.v1 import LFP, ImportedLFP
+
+schema = dj.schema("lfp")
+
+
+@schema
+class LFPOutput(dj.Manual):
+    definition = """
+    lfp_id: uuid
+    ---
+    stream: varchar(40)
+    """
+
+    class LFP(dj.Part):
```

Comment:
This is where we added versioning, so DLCPosV1, TrodesPosV1, and CommonPos (assuming that common_position is deprecated and will not be versioned). This way you can later add a part table LFPv2 without conflict.